### DISCUSSION TITLE
Convert on budget account to an off budget account

### DISCUSSION THREAD

**Discussion by bpaulien** [actualbudget/releases#83] [Q&A] [answered]
Keeping all the transaction history intact, I'd like to be able to move an account Off -> On budget, or from On -> Off budget.  Maybe that's not possible.  Is there an easy work around, if it can't be done?  I really don't want to lose the transaction history.

to be honest, since I just started, I only  have a couple of transactions on this card, so it wouldn't be the worst thing to lose, but still, looking for ideas.

**Comment by MikeBishop** [comment, marked as answer]
For the budget to stay accurate, that money needs to leave the budget.  Seems like the simplest approach would be to create a new off-budget account, transfer the full balance across (accounting for that outflow in your budget) and then close the on-budget account.  The history is still accessible via the now-closed version of the account.
